src/deploy/NVA_build/Tests.Dockerfile

```diff
@@ -1,21 +1,92 @@
-FROM centos:7
+FROM centos:7 as base
 LABEL maintainer="Liran Mauda (lmauda@redhat.com)"
 
 ######################
 # NOOBAA SETUP TESTS #
 ######################
 
 ENV container docker
-COPY /noobaa-core/ .
-RUN yum install -y which && \
+ENV TEST_CONTAINER true
```

Comment:
Can you move this to the `unittest` image?
I think it doesn't belong to `base` as it should become the base builder image.